### PR TITLE
[6] Product Page using HTML

### PR DESCRIPTION
**Pull request by Olawumi11**
## Task: `Product page using HTML`
- `added an HTML file for an e-commerce site product page'

### CODE REVIEW

File: submissions/famuyiwaOlawumi/HTML/index.html [hunk 1]
```diff
@@ -0,0 +1,170 @@
+<!DOCTYPE html>
```
Comment (ToluAkin) [inline submissions/famuyiwaOlawumi/HTML/index.html:1]: Well done! You are require to use ONLY 2 product images. Delete the images that you are not using.

File: submissions/famuyiwaOlawumi/HTML/index.html [hunk 2]
```diff
@@ -0,0 +1,170 @@
...
+<html>
+<head>
+        <meta charset="UTF-8">
+        <title>xyzmart</title>
+</head>
+
```
Comment (ToluAkin) [inline submissions/famuyiwaOlawumi/HTML/index.html:7]: Remove the extra one or 2 lines spaces in this file.

File: submissions/famuyiwaOlawumi/HTML/index.html (hunk 1 above)
Comment (Olawumi11) [reply submissions/famuyiwaOlawumi/HTML/index.html:1]: Can I get more clarification on this? 
The first two images I inserted is for the actual product of interest. The others is for similar products. Are you saying I don't need images to depict the section for similar products the buyer might like? Or am I to give 1 image to the actual product of interest and then 1 to similar products of interest?
Thanks

File: submissions/famuyiwaOlawumi/HTML/index.html (hunk 2 above)
Comment (Olawumi11) [reply submissions/famuyiwaOlawumi/HTML/index.html:7]: Ok

File: submissions/famuyiwaOlawumi/HTML/index.html (hunk 1 above)
Comment (ToluAkin) [reply submissions/famuyiwaOlawumi/HTML/index.html:1]: In the instructions for this task, you were asked to use only 2 product images but you have 6 images. Remove the extra images and delete the unused ones. Check [here](https://github.com/Dufuna-CodeCamp/Dufuna-CodeCamp20/issues/7) for the instruction. 
Yes, you don't need images to depict the similar products.

**Pull request closed without merging**